### ISSUE TITLE
🐛 Bug Report: remove @backstage/backend-common from dependencies in @backstage/backend-defaults

### ISSUE THREAD

**Issue by richard-hunter-hpe** [backstage/backstage#27688]
### 📜 Description

@backstage/backend-common Fails CVE scans as it has a dependency on `@kubernetes/client-node` which cannot be updated as it is pinned to a version. If it has been removed from source code as mentioned here: https://backstage.io/docs/releases/v1.32.0#promoted-new-backend-system-exports-to-default-export then it would be good to remove it from `package.json` https://github.com/backstage/backstage/blob/994506952c20d7708c2f8a5f6498b19132157166/packages/backend-defaults/package.json#L123

```
└─┬ backend@0.0.0 -> ./packages/backend
  └─┬ @backstage/backend-defaults@0.5.2
    └─┬ @backstage/backend-common@0.25.0
      └─┬ @kubernetes/client-node@0.20.0
        └── jsonpath-plus@7.2.0 deduped
```

```
  },
  "dependencies": {
    "@aws-sdk/abort-controller": "^3.347.0",
    "@aws-sdk/client-codecommit": "^3.350.0",
    "@aws-sdk/client-s3": "^3.350.0",
    "@aws-sdk/credential-providers": "^3.350.0",
    "@aws-sdk/types": "^3.347.0",
    "@backstage/backend-dev-utils": "^0.1.5",
    "@backstage/backend-plugin-api": "^1.0.0",
    "@backstage/cli-common": "^0.1.14",
    "@backstage/config": "^1.2.0",
    "@backstage/config-loader": "^1.9.1",
    "@backstage/errors": "^1.2.4",
    "@backstage/integration": "^1.15.0",
    "@backstage/integration-aws-node": "^0.1.12",
    "@backstage/plugin-auth-node": "^0.5.2",
    "@backstage/types": "^1.1.1",
    "@google-cloud/storage": "^7.0.0",
    "@keyv/memcache": "^1.3.5",
    "@keyv/redis": "^2.5.3",
    "@kubernetes/client-node": "0.20.0" <--- can't update this using yarn up -R
...
```

### 👍 Expected behavior

If the source has wiped reference to the package, it should be removed from dependencies

### 👎 Actual Behavior with Screenshots

Fails CVE scans due to https://avd.aquasec.com/nvd/cve-2024-21534 

### 👟 Reproduction steps

```trivy fs --exit-code 1 --severity HIGH,CRITICAL --no-progress --skip-dirs node_modules .```

### 🖥️ Your Environment

ubuntu 20

### 👀 Have you spent some time to check if this bug has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/backstage/backstage/blob/master/CODE_OF_CONDUCT.md)

### Are you willing to submit PR?

Yes I am willing to submit a PR!

**Comment by vinzscam** [comment]
Thanks for pointing it out @richard-hunter-hpe! Opened https://github.com/backstage/backstage/pull/27698

**Comment by benjdlambert** [comment]
Closing as fixed in #27698
